Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,14 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const inHours = tasks.map(hour => hour['duration']/60);
+console.log(inHours);
+
+const moreThanTwo = inHours.filter(hour => hour > 2);
+console.log(moreThanTwo);
+
+const amountPerHour = moreThanTwo.map(hour => hour * 15);
+console.log(amountPerHour);
+
+const  roundedToEuro = amountPerHour.toLocaleString('it-IT', {style: 'currency', currency: 'EUR'});
```

Comment:
For this assignment we are only interested in a single number that represents the total earnings for Maartje in Euros. Here you are apparently trying to report a payment for each individual task and leave it to us to add it all up. Besides the formatting in Euros doesn't seem to work. And why did you choose Italy ('it-IT') as the locale?

Reply:
Thanks for feedback , i have tried tofixed(2) but its did not works for me then i found this on google .toLocaleString('it-IT', {style: 'currency', currency: 'EUR'}); and it works just fine in the browser , but now i have fixed all what u have mentioned.

Reply:
Have you also tried `toFixed(2)` (mind the exact spelling)? If you google and find something that seems to work but that you can't explain, then you shouldn't use it. It is not good enough if it works without you knowing why.